server/data-base/inventory.json

```diff
@@ -5,22 +5,50 @@
   "novelties": [
     {
       "category": "Shirts and Sweaters",
-      "items": []
+      "items": [
+        {
+          "item": "Cheat Sheet Tee",
+          "price": "$20.00",
+          "description": "Available in S/M/L/XL",
+          "image": "/assets/img/inventory/novelties/shirts/cheatsheet.jpg"
+        }
+      ]
     },
     {
       "category": "Stickers",
-      "items": []
+      "items": [
+        {
+          "item": "Laptop Fire Cat Sticker",
+          "price": "$0.25",
+          "description": null,
+          "image": "/assets/img/inventory/novelties/stickers/laptopfirecat.jpg"
+        }
+      ]
     },
     {
       "category": "Ties",
-      "items": []
+      "items": [
+        {
+          "item": "2022 Pink Tie",
+          "price": "$2.00",
+          "description": null,
```

Comment:
these `null`s appear to be breaking the backend - thoughts on changing them to `""`?